doc/man7/EVP_SIGNATURE-ED25519.pod

```diff
@@ -64,16 +120,24 @@ specified, then both Ed25519 and Ed448 are benchmarked.
 
 =head1 EXAMPLES
 
-To sign a message using a ED25519 or ED448 key:
+To sign a message using an ED25519 EVP_PKEY structure:
 
     void do_sign(EVP_PKEY *ed_key, unsigned char *msg, size_t msg_len)
     {
         size_t sig_len;
         unsigned char *sig = NULL;
         EVP_MD_CTX *md_ctx = EVP_MD_CTX_new();
 
-        EVP_DigestSignInit(md_ctx, NULL, NULL, NULL, ed_key);
-        /* Calculate the requires size for the signature by passing a NULL buffer */
+        const OSSL_PARAM params[] = {
+            OSSL_PARAM_utf8_string ("instance", "Ed25519ctx", 10),
```

Comment:
you can specify 0 instead of 10 here..

Reply:
cool.  I didn't know that.

I think I will keep it as "10" so people reading the doc don't have to jump to OSSL_PARAM_int.html to figure out why "0" is okay.